src/pages/hegel/guides/general-division-of-the-logic.mdx

```diff
@@ -0,0 +1,101 @@
+
+Third, the objectification of the `I`, or what could be called the real process whereby the `I` becomes a representation for itself, can be understood as an "original and necessary deed of consciousness" (Hegel, 2010, 41/21.47) such that this essentially precedes the subject-object dichotomy integral to consciousness. But then this should not be called consciousness anymore, Hegel says, since consciousness is explicitly that for which "holds within itself the opposition of the 'I', and its intended object which is not to be found in that original deed" (Hegel, 2010, 41/21.47-48). Where this opposition has ceased the matter should not be termed consciousness but thought.
+
+We see here that for all the shortcomings of Kant's system (if Hegel is right), there are key links in the manner both these philosophers build their systems. Where Kant essentially begins with what is most immediate to consciousness&mdash;pure intuitions of space and time&mdash;Hegel refines these categories further to those that concern immediacy pure and simple, such as `being` and `nothing`. Gone is the presupposition of an empirical reality where the examination of pure thought is concerned, there is only, minimally, the immediate being of that thought&mdash;which is exactly _not_ external to that thought (see also [Presuppositionless Thinking](/hegel/guides/presuppositionless-thinking)). Now, this is not to deny that there is such a thing as empirical reality, but following Hegel's systematic development, whatever empirical reality turns out to be, it cannot be less than immediate being, which _also_ overlaps with thought, such that this category is shared between empirical reality and thought. This solves the problem of mediation between thought and being, or subjectivity and objectivity, since from the start the least that empirical reality can be is also minimally conceptual or logical.
```

Comment:
"it cannot be less than immediate being, which _also_ overlaps with thought," - is this something that you touch on in the introduction article? I can't recall. I ask because it's kind of a big thing to just casually throw in there.

Reply:
Hmm, yes, it maps on to the issue of thought and being, or logic and metaphysics. I added a link to that section in the introduction!